src-tauri/src/main.rs

```diff
@@ -14,8 +15,13 @@ async fn login() -> String {
 }
 
 #[tauri::command]
```

Comment:
Not asking for any changes here, but in the future can you ensure that diffs are more legible? All you're doing here is adding `pay_invoice` method and removing some empty lines in `sign_event`, but that's not entirely clear at a glance. If you declared `pay_invoice` below `sign_event`, I think this would have been easier to grok.